CHANGELOG.md

```diff
@@ -22,11 +22,22 @@ Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
 
+## [2.25.2] - 2022-01-21
+[2.25.2]: https://github.com/atlassian/aws-infrastructure/compare/release-2.25.1...release-2.25.2
+
 ### Fixed
 - Grant Jira access to its own HTTP port in `StandaloneFormula`. Fix [JPERF-790].
 
 [JPERF-790]: https://ecosystem.atlassian.net/browse/JPERF-790
 
+## [2.25.1] - 2022-01-14
+[2.25.1]: https://github.com/atlassian/aws-infrastructure/compare/release-2.25.0...release-2.25.1
+
+### Fixed
+- Roll back change to one of `DataCenterFormula`'s log messages done accidentally as part of [PR#121].
+
+[PR#121]: https://github.com/atlassian/aws-infrastructure/pull/121
```

Comment:
Is this link to PR good for CHANGELOG?

Reply:
They are stored forever.
But changelog is for consumers. Consumers don't operate on pull requests, but on versions.

Reply:
> But changelog is for consumers. Consumers don't operate on pull requests, but on versions.

I'm open for any suggestions.

Reply:
This case is specific, because the release was only to correct the internal details of the code. Not sure if log message is API in any way.

Reply:
```
- Roll back a `DataCenterFormula` log message to 2.25.0
```

True, undocumented behavior is not API, so it has no SemVer consequences. But we do document bug fixes, because it affects consumers so they can reason about bumps/downgrades.

Reply:
Ok, I see. I will this message a bit differently if you don't mind:

> - Roll back change made to one of `DataCenterFormula`'s log messages done accidentally as part of [2.25.0].